.strigo/config.yml

```diff
@@ -0,0 +1,7 @@
+# Classes exercises configuration
+---
+classes:
+  - class_id: <your_class_id>
```

Comment:
Also also, I'd comment that while this is part of the spec, it doesn't currently do anything. WDYT?

Reply:
It will be used from day 1. We will only retrieve exercises for a given class id (i.e. for class we are trying to edit).

Reply:
Um.. I don't think it should be a requirement. What if they're using a build process to create a class and then update it via the api automatically to include exercises from GitHub? How will they update the class id in the yaml?

Reply:
They'll need to POST to `classes/:classId/exercises` so anyway, they must have a class id at this point. If we omit the `class_id` here, we won't be able to share files between multiple classes.

Reply:
Um.. I'm continuing it here. 

I don't think that they **must** include a class ID. They could potentially create the class as part of their CI process (like VMware does). In that case, all it means is that they call our API to create a class, and then call the API using the returned class ID to add a Github repo and fetch the exercises.

Reply:
So maybe instead of having a multiple classes' config we should use a config file per course? IDK, this makes more sense to me to have:
```yaml
# my-awesome-class-config.yml
---
exercises:
  - file: x
  - file: y
```
WDYT?